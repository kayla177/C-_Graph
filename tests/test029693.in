ENTITY 6UIIKMBE name_6UII journal
ENTITY DXBNPN78 name_DXBN author
ENTITY XQ8GICO6 name_XQ8G paper
ENTITY TX5LOYUW name_TX5L author
ENTITY EQI6VKMW name_EQI6 paper
ENTITY 5HXNTPNO name_5HXN paper
ENTITY T8PE6BO5 name_T8PE conference
ENTITY FKBHMA52 name_FKBH author
ENTITY GD6C2IKK name_GD6C author
ENTITY OLCUCOQT name_OLCU paper
RELATIONSHIP DXBNPN78 presented_at GD6C2IKK 4.5
RELATIONSHIP GD6C2IKK presented_at OLCUCOQT 1.6
RELATIONSHIP DXBNPN78 collaborated_with T8PE6BO5 7.2
RELATIONSHIP DXBNPN78 collaborated_with FKBHMA52 7.1
RELATIONSHIP FKBHMA52 cited_by 5HXNTPNO 4.8
RELATIONSHIP 6UIIKMBE presented_at DXBNPN78 9.0
RELATIONSHIP T8PE6BO5 authored_by EQI6VKMW 7.0
RELATIONSHIP 6UIIKMBE affiliated_with GD6C2IKK 6.1
RELATIONSHIP FKBHMA52 affiliated_with T8PE6BO5 1.2
RELATIONSHIP FKBHMA52 affiliated_with GD6C2IKK 7.6
RELATIONSHIP 5HXNTPNO cited_by EQI6VKMW 0.7
RELATIONSHIP 5HXNTPNO affiliated_with TX5LOYUW 0.5
RELATIONSHIP DXBNPN78 affiliated_with EQI6VKMW 0.4
RELATIONSHIP GD6C2IKK collaborated_with XQ8GICO6 5.6
RELATIONSHIP 6UIIKMBE cited_by TX5LOYUW 1.4
DELETE EQI6VKMW
DELETE TX5LOYUW
DELETE 6UIIKMBE
HIGHEST
PRINT 5HXNTPNO
HIGHEST
HIGHEST
HIGHEST
HIGHEST
HIGHEST
EXIT
