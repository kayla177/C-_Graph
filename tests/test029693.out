success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
5HXNTPNO T8PE6BO5 19.1
FKBHMA52
5HXNTPNO T8PE6BO5 19.1
5HXNTPNO T8PE6BO5 19.1
5HXNTPNO T8PE6BO5 19.1
5HXNTPNO T8PE6BO5 19.1
5HXNTPNO T8PE6BO5 19.1
